test/functional/feature_coinstatsindex.py

```
@@ -313,6 +316,26 @@ def _test_index_rejects_hash_serialized(self):
         for use_index in {True, False, None}:
             assert_raises_rpc_error(-8, msg, self.nodes[1].gettxoutsetinfo, hash_type='hash_serialized_2', hash_or_height=111, use_index=use_index)
 
+    def _test_init_index_after_reorg(self):
```

Ruff (0.12.2)

Missing return type annotation for private function `_test_init_index_after_reorg`

Add return type annotation: `None`

(ANN202)